CMakeLists.txt

```diff
@@ -80,6 +80,14 @@ find_package(USD REQUIRED)
 include(cmake/usd.cmake)
 include(${USD_CONFIG_FILE})
 
+string(SUBSTRING ${MAYA_API_VERSION} "0" "4" MAYA_MAJOR_VERSION)
+if(${MAYA_MAJOR_VERSION} STRLESS "2019")
+    set(CMAKE_WANT_UFE_BUILD OFF)
+    message(AUTHOR_WARNING "========================================================================================= \
+                            UFE is not available in Maya${MAYA_MAJOR_VERSION}. Maya2019 and later are supported only. \
+                            ========================================================================================= ")
+endif()
+
```

Comment:
If you end up moving your stuff to FindMaya.cmake I would suggest moving this up to directly under line 33.

Reply:
Hey @seando-adsk,  See 766736c 

> If you end up moving your stuff to FindMaya.cmake I would suggest moving this up to directly under line 33.

This won't work because we want to have this check after `find_package(Maya REQUIRED)` is called.

Reply:
As yes you are correct.